src/main/java/de/tum/in/www1/artemis/repository/CourseRepository.java

```diff
@@ -124,11 +130,12 @@
     String getCourseTitle(@Param("courseId") Long courseId);
 
     @Query("""
-            select distinct c
-            from Course c left join fetch c.exercises e
-            where c.instructorGroupName in :#{#userGroups} and TYPE(e) = QuizExercise
+            SELECT DISTINCT c
+            FROM Course c LEFT JOIN FETCH c.exercises e
+            WHERE (c.instructorGroupName IN :#{#userGroups} OR c.editorGroupName IN :#{#userGroups})
```

Comment:
Is it intentional that you check if the user groups are inside the instructor group? Shouldn't you only check in the editor group?

Reply:
No, we wanted both groups. But I renamed the method to "...AtLeastEditorAccess"